_posts/2021-12-31-trino-2021-a-year-of-growth.md

```diff
@@ -0,0 +1,348 @@
+---
```

Comment:
I would add a "by the numbers" section somewhere that just shows a highlight of all the quantifiable metrics: slack members, commits, TCB episodes, youtube/twitter followers, releases, etc.